### PR TITLE
fix: Prevent the binary from getting checked in when using go build

### PR DESCRIPTION
**Pull request by hilts-vaughan**
## Change Description

I've almost checked the binary a few times in; is there a reason we need to have this in the repo? 

## Checklist

- [x] Make sure to open an issue as a 
  [bug/issue](https://github.com/GoogleCloudPlatform/cloudsql-proxy/issues/new/choose) 
  Small change; can open an issue if there's enough of a reason to
- [x] Ensure the tests and linter pass
- [x] Appropriate documentation is updated (if necessary)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,7 @@
 # Jetbrains IDE
 .idea/
 
-tests/cloud_sql_proxy
+# Binary output
+cmd/cloud_sql_proxy/cloud_sql_proxy
+
+tests/cloud_sql_proxy
```
Comment (kurtisvg) [inline .gitignore:8]: nit: Probably best to leave a newline at the end

**Comment by kurtisvg** [comment]
@hilts-vaughan I generalized a bit. Does that look okay to you?

**Comment by hilts-vaughan** [comment]
I generally would prefer to be more explicit but I guess given you can build from any directory, that's probably fine. :)

Your project! It meets my use case.

**Comment by kurtisvg** [comment]
> I generally would prefer to be more explicit but I guess given you can build from any directory, that's probably fine. :)
> 
> Your project! It meets my use case.

So I would have just done `cmd/cloud_sql_proxy/cloud_sql_proxy`. Could you expand why you wanted to ignore `tests` too? Do you usually copy the binary in there?

**Comment by hilts-vaughan** [comment]
I'm confused. It's ignored in `tests` in `master` right now. 

You have proposed to change to `*/cloud_sql_proxy`

I just added a line.

**Comment by kurtisvg** [comment]
Your right, I looked at the diff wrong

**Pull request merged**